distance3d/hydroelastic_contact/_broad_phase.py

```diff
@@ -33,30 +33,35 @@ def broad_phase_tetrahedra(rigid_body1, rigid_body2):
     aabbs2 = tetrahedral_mesh_aabbs(rigid_body2.tetrahedra_points)
 
     # TODO fix broad phase for cube vs. sphere
+    # Result: The AABBs of an axis aligned Cube contains always two equal AABBs for every side of the cube.
+    # when unique is turned on the AABBs library deletes one pair hence the wrong results.
+
     # TODO speed up broad phase with numba
     # TODO store result in RigidBody
 
-    """
-    tree2 = aabbtree.AABBTree()
-    for j, aabb in enumerate(aabbs2):
-        tree2.add(aabbtree.AABB(aabb), j)
-    broad_tetrahedra1 = []
-    broad_tetrahedra2 = []
-    for i, aabb in enumerate(aabbs1):
-        new_indices2 = tree2.overlap_values(aabbtree.AABB(aabb))
-        broad_tetrahedra2.extend(new_indices2)
-        broad_tetrahedra1.extend([i] * len(new_indices2))
-
-    broad_pairs = list(zip(broad_tetrahedra1, broad_tetrahedra2))
-    assert len(broad_tetrahedra1) == len(broad_tetrahedra2)
-    for i, aabb in enumerate(aabbs1):
-        if tree2.does_overlap(aabbtree.AABB(aabb)):
-            assert i in broad_tetrahedra1
-        else:
-            assert i not in broad_tetrahedra1
-    """
-
-    return _all_aabbs_overlap(aabbs1, aabbs2)
+    test_AABBs_Tree = False
```

Comment:
Please don't mix snake and camel case. My suggestion is: `test_aabb_tree`.

Anyway, in the end there should either only be one solution or it should be a parameter of the public function.